### ISSUE TITLE
n-ML: Mitigating Adversarial Examples via Ensembles of Topologically Manipulated Classifiers

### ISSUE THREAD

**Issue by Apromixately** [robust-ml/robust-ml.github.io#16]
**Name**: n-ML: Mitigating Adversarial Examples via Ensembles of Topologically Manipulated Classifiers
**Authors**: Mahmood Sharif, Lujo Bauer, Michael K. Reiter

**Paper**: https://arxiv.org/pdf/1912.09059.pdf

**Code**: -

**Venue**: -

**Does the code implement the robust-ml API and include pre-trained models**: -

**Dataset**: MNIST, CIFAR10, GTSRB

**Threat model**: white box, gray box, black box

**Natural accuracy**: e.g. CIFAR10 / black box / L_inf <= 8/255: 94.50 %

**Claims**: e.g. CIFAR10 / black box / L_inf <= 8/255: 100.00 %

**Comment by anishathalye** [comment]
Thank you for the submission! (Are you one of the authors? I can't guess from your GitHub profile.)

As per our [policy](https://www.robust-ml.org/faq/#how-are-new-defenses-added-to-the-list), in order to be listed, defenses must have code publicly available (with pre-trained models, and implementing the robust-ml API). Do you know if there's code available for this defense?

**Comment by Apromixately** [comment]
I am not one of the authors. I've sent an email to Mahmood last week but haven't gotten a reply, yet.

**Comment by Apromixately** [comment]
Oh, in case you haven't had a look: they say in the paper that they will publish the code when publishing the paper. So it might just not have happened yet or be postponed until it is accepted at a conference.
